docker/etc/rest-source-authorizer/authorizer.yml

```diff
@@ -19,10 +19,14 @@ database:
   password: radarcns
   dialect: org.hibernate.dialect.PostgreSQLDialect
 
+redis:
+  uri: redis://redis:6379
+
 restSourceClients:
-  - sourceType: FitBit
-    authorizationEndpoint: https://www.fitbit.com/oauth2/authorize
-    tokenEndpoint: https://api.fitbit.com/oauth2/token
-    clientId: Fitbit-clientid
-    clientSecret: Fitbit-clientsecret
-    scope: activity heartrate sleep profile
+  - sourceType: Oura
```

Comment:
I think we can just keep `Fitbit` here as well? Just like in the `authorizer.yml.template`.

Reply:
Sorry @Hsankesara, I meant can we keep both Fitbit and Oura here?

Reply:
Ah sorry, my bad. That makes more sense. Will update it